util/linuxfw/nftables.go

```diff
@@ -129,6 +129,29 @@ func detectNetfilter() (int, error) {
 		}
 		validRules += len(rules)
 	}
+	if validRules > 0 {
+		// We have valid rules, so we can use nftables.
+		return validRules, nil
+	}
+
+	// We could not find any valid rules, so try creating a dummy postrouting
+	// chain. Emperically, we have noticed that on some devices there is partial
+	// nftables support and the kernel rejects some chains that are valid on
```

Comment:
mention GKE? Or reference an issue with details?

Otherwise this will always be scary and nobody will want to change this later, even if GKE changes.